src/main/java/com/hartwig/platinum/kubernetes/PipelineJob.java

```diff
@@ -49,6 +53,9 @@ public JobSpec asKubernetes() {
         }
         JobSpec spec = dsl.endSpec().endTemplate().build();
         spec.setAdditionalProperty("backoffLimit", 1);
+        if (!ttl.equals(Duration.ZERO)) {
```

Comment:
I think something like `Duration.NOT_SET` would be more clear, but `ZERO` also works of course.